### PR TITLE
report: make denser. changes to typography, group descriptions, more

### PR DESCRIPTION
**Pull request by paulirish**
Quite a few changes in here. I've been iterating with jiwoong over the past week. 

Currently draft, but will probably flip in a few.

### CODE REVIEW

**Comment by adamraine** [comment]
Average gauge text looks really red in dark mode:

Screen Shot 2021-10-29 at 2 04 52 PM

File: report/assets/styles.css [hunk 1]
```diff
@@ -50,20 +51,21 @@
...
+  --color-orange: #fa3;
+  --color-red-700: #c00;
+  --color-red: #f33;
   --color-teal-600: #00897B;
   --color-white: #FFFFFF;
 
   /* Context-specific colors */
   --color-average-secondary: var(--color-orange-700);
   --color-average: var(--color-orange);
+  --color-average-text: #C33300;
```
Comment (adamraine) [inline report/assets/styles.css:68]: `--color-orange-700` instead?

Comment (paulirish) [reply report/assets/styles.css:68]: jiwoong selected this instead.

File: report/test/renderer/report-ui-features-test.js [hunk 2]
```diff
@@ -92,7 +92,7 @@ describe('ReportUIFeatures', () => {
       assert.equal(dom.findAll('.lh-category', container).length, 1);
     });
 
-    describe('third-party filtering', () => {
```
[the remark this answers is not in the document]
Comment (paulirish) [reply report/test/renderer/report-ui-features-test.js:95]: nice catch.

**Comment by paulirish** [comment]
> Average gauge text looks really red in dark mode:

Yup. Known issue.

**Comment by adamraine** [comment]
Some links are missing the correct color:

Screen Shot 2021-11-01 at 6 13 45 PM

File: report/assets/styles.css [hunk 3]
```diff
@@ -75,81 +77,80 @@
...
   --report-icon-size: var(--score-icon-background-size);
   --report-line-height: 24px;
-  --report-min-width: 400px;
+  --report-line-height-secondary: 20px;
+  --report-min-width: 360px;
   --report-monospace-font-size: calc(var(--report-font-size) * 0.85);
   --report-text-color-secondary: var(--color-gray-800);
   --report-text-color: var(--color-gray-900);
-  --report-width: calc(60 * var(--report-font-size));
+  --report-max-width: calc(60 * var(--report-font-size));
```
Comment (adamraine) [inline report/assets/styles.css:131]: This is used once in the flow `styles.css`

File: report/assets/styles.css [hunk 4]
```diff
@@ -370,6 +371,10 @@
   color: var(--color-informative);
 }
 
+.lh-audit-group a {
+  color: var(--color-blue-primary);
```
Comment (adamraine) [inline report/assets/styles.css:375]: I see the link color is intentional, but we have two different link colors now. Should this color be used report-wide?

File: report/assets/styles.css [hunk 5]
```diff
@@ -1025,14 +975,15 @@
   color: var(--color-red-700);
 }
 
+
```
Comment (adamraine) [inline report/assets/styles.css:978]: nit
```suggestion
```

File: report/assets/styles.css [hunk 6]
```diff
@@ -1058,61 +1009,46 @@
...
+  line-height: var(--report-line-height-secondary);
+  color: var(--color-gray-700);
 }
 
-.lh-audit-group__header span.lh-audit-group__itemcount {
-  font-weight: bold;
-  color: var(--color-gray-600);
+.lh-audit-group__title {
+  text-transform: uppercase;
+	font-weight: 500;
```
Comment (adamraine) [inline report/assets/styles.css:1033]:
```suggestion
  font-weight: 500;
```

File: report/assets/styles.css [hunk 7]
```diff
@@ -1058,61 +1009,46 @@
...
-   It's more straightforward to hide w/ CSS vs change categoryRenderer.renderAuditGroup for this case. */
-.lh-audit-group--metrics > .lh-audit-group__header {
-  /* Also, it's vis hidden (and not display:none) because of spacing regarding the pill. */
-  visibility: hidden;
+.lh-details,
+.lh-category-header__description,
+.lh-load-opportunity__header,
+.lh-audit-group__footer {
+	font-size: var(--report-font-size-secondary);
+	line-height: var(--report-line-height-secondary);
```
Comment (adamraine) [inline report/assets/styles.css:1051]:
```suggestion
  font-size: var(--report-font-size-secondary);
  line-height: var(--report-line-height-secondary);
```

File: report/assets/styles.css [hunk 8]
```diff
@@ -1200,19 +1139,19 @@
...
 .lh-gauge__wrapper--pass {
-  color: var(--color-pass);
+  color: var(--color-pass-secondary);
   fill: var(--color-pass);
   stroke: var(--color-pass);
 }
 
 .lh-gauge__wrapper--average {
-  color: var(--color-average);
+  color: var(--color-average-text);
```
Comment (adamraine) [inline report/assets/styles.css:1148]: Why not name this `--color-average-secondary` to match the others?

File: report/assets/styles.css [hunk 9]
```diff
@@ -1689,23 +1633,53 @@
...
+  border-bottom: 1px solid var(--report-border-color-secondary);
+}
+details.lh-clump[open] > summary {
+  border-bottom-width: 0;
+}
+
+
+
+details .lh-clump-toggle::before {content: 'Show';}
+details[open] .lh-clump-toggle::before {content: 'Hide';}
```
Comment (adamraine) [inline report/assets/styles.css:1682]: Some more strings that could be localized here

File: report/assets/styles.css [hunk 10]
```diff
@@ -1820,20 +1786,23 @@
 
 
 .lh-meta__items {
+  --meta-icon-size: calc(var(--report-icon-size) * 0.667);
```
Comment (adamraine) [inline report/assets/styles.css:1789]: Does this need to be full width? I kinda liked it when it was narrower + centered.

File: report/assets/styles.css [hunk 11]
```diff
@@ -1153,9 +1081,13 @@
 
 .lh-warnings {
   --item-margin: calc(var(--report-line-height) / 6);
```
Comment (adamraine) [inline report/assets/styles.css:1083]: This is also takes up the full width where it didn't before.

**Review by adamraine — APPROVED** [review]
Overall LGTM

File: report/assets/styles.css [hunk 12]
```diff
@@ -779,62 +773,31 @@
...
-}
-.lh-metrics-toggle__lines {
-  fill: var(--metric-toggle-lines-fill);
+  top: 0px;
 }
 
-.lh-metrics-toggle__label  {
-  background-color: var(--metrics-toggle-background-color);
+.lh-metrics-toggle__input + div > label::before {content: 'Expand view';}
+.lh-metrics-toggle__input:checked + div > label::before {content: 'Collapse view';}
```
[the remark this answers is not in the document]
Comment (paulirish) [reply report/assets/styles.css:778]: pulled into separate PR https://github.com/GoogleChrome/lighthouse/pull/13308

File: report/assets/styles.css (hunk 9 above)
Comment (paulirish) [reply report/assets/styles.css:1674]: pulled into separate PR https://github.com/GoogleChrome/lighthouse/pull/13308

**Pull request merged**